docker/emptydrops/emptyDropsWrapper/test/prepRDS.R

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env Rscript
+
+library('optparse')
+
+option_list <- list(
+	    make_option(c('-i','--input'),
```

Comment:
nit: indentation styles

Reply:
Can you please be more specific? Note R doesn't care about whitespace it's only cosmetic.

Reply:
Nothing wrong here, I just figured the `make_option` are not aligned with each other. It's trivial style nit

Reply:
I got it it wasn't clear on the diff, but was on the source